tests/integration/strategy_tests.cpp

```diff
@@ -119,32 +119,21 @@ TEST_P(StrategyTest, Duplicate)
 
 INSTANTIATE_TEST_CASE_P(Allocations, StrategyTest, ::testing::ValuesIn(AllocationDevices),);
 
-#if defined(UMPIRE_ENABLE_DEVICE)
-TEST(Strategy, Device)
+TEST(DynamicPool, LimitedResource)
 {
   auto& rm = umpire::ResourceManager::getInstance();
 
-  auto allocator = rm.getAllocator("DEVICE");
-  void* alloc = nullptr;
+  const std::size_t max_mem{1024 * 1024 * 4};
 
-  // Determine how much memory we can allocate from device
```

Comment:
Why are we throwing this out?

Reply:
This part was integral for the test, to make the allocator `throw` internally further down. I replaced it by a `SizeLimiter` strategy, so we don't need this part any more.